plugins/processors/awsappsignals/config/config.go

```diff
@@ -34,10 +34,11 @@ const (
 
 func NewDefaultLimiterConfig() *LimiterConfig {
 	return &LimiterConfig{
-		Threshold:         DefaultThreshold,
-		Disabled:          false,
-		LogDroppedMetrics: false,
-		RotationInterval:  DefaultRotationInterval,
+		Threshold:                 DefaultThreshold,
+		Disabled:                  false,
+		LogDroppedMetrics:         false,
+		RotationInterval:          DefaultRotationInterval,
+		GarbageCollectionInterval: DefaultGCInterval,
```

Comment:
Hmmm, interesting, I thought`GarbageCollectionInterval` will be assigned with `DefaultGCInterval` in the `Validate()` if no custom value is provided.

Does the test shows that after assigning a default value, the CPU usage drops?

Reply:
Do you mean the config `Validate()` function? That's called by the OTEL collector on start up (https://github.com/open-telemetry/opentelemetry-collector/blob/8990be35990b9833ad98d9bc254667560b7a9951/otelcol/collector.go#L182). We really shouldn't be relying on the Validate function to set defaults in the config.